tools/scripts/benchmark.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```

Comment:
Are you planning to publish the benchmark.sh results in a CI job like the markdown you posted in the PR description?

Reply:
Yes that's the plan. Currently busy rebuilding my on-site infrastracture, I will eventually provide a dedicated jenkins node to run the benchmark.